src/transformers/models/deberta_v2/modeling_deberta_v2.py

```diff
@@ -782,7 +786,7 @@ def disentangled_attention_bias(self, query_layer, key_layer, relative_pos, rel_
 
         # position->content
         if "p2c" in self.pos_att_type:
-            scale = math.sqrt(pos_query_layer.size(-1) * scale_factor)
+            scale = torch.sqrt(torch.tensor(pos_query_layer.size(-1) * scale_factor, dtype=torch.float))
```

Comment:
Just to remark that with `torch.tensor`, `scale` will be recognized as a constant, so both ways shall have the same effect. But I do agree to replace math with torch.

Reply:
I agree, altough I am not sure, I would be more confortable with:
```suggestion
            scale = torch.sqrt(torch.tensor(pos_query_layer.size(-1), dtype=torch.float) * scale_factor)
```